pkg/services/ngalert/notifier/channels/testing.go

```diff
@@ -142,3 +148,27 @@ func (ns *notificationServiceMock) SendEmailCommandHandler(ctx context.Context,
 }
 
 func mockNotificationService() *notificationServiceMock { return &notificationServiceMock{} }
+
+func CreateNotificationService(t *testing.T) *notifications.NotificationService {
+	t.Helper()
+
+	tracer, err := tracing.InitializeTracerForTest()
```

Comment:
This line caused the merge conflict.  InitializeTracerForTest() was changed to return an error as well.